config-connector/solutions/projects/kpt/owned-project/README.md

```diff
@@ -0,0 +1,66 @@
+Owned Project
+==================================================
+
+# NAME
+
+  owned-project
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create
+  a project in a folder, binding an IAM member
+  as project owner.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  Run `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/projects/kpt/project-owner project-owner`.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a
+  service account with the following roles in
+  the organization:
+  - roles/resourcemanager.folderViewer
+  - roles/resourcemanager.projectCreator
+  - roles/iam.securityAdmin
+
+# USAGE
+
+  Replace the
+  `${BILLING_ACCOUNT_ID?}` value:
```

Comment:
The colon in the end makes me feel that there is a command missing. Let's not do that.